api/pom.xml

```diff
@@ -54,6 +54,13 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>airline</artifactId>
+            <exclusions>
+                <exclusion>
+                    <!--GPL licenced library-->
+                    <groupId>com.google.code.findbugs</groupId>
```

Comment:
Does defining the exclusion at the place we define the version work? it would be nice if future extensions that pull in airlift for some reason don't accidentally pull in a license that is not expected.